### DISCUSSION TITLE
讨论计划明年将 Natasha 文档切换到英文

### DISCUSSION THREAD

**Discussion by NMSAzulX** [dotnetcore/Natasha#44] [General]
由于今年 Natasha issue 区并不活跃，包括交流群，并未得到更多的反馈，我甚至不清楚有多少人在使用，让我一直认为 Natasha 已经接近完善。为了寻找更多的建议和新特性，我打算明年切换到英文状态，面向国际友人，同时将中文文档转移到博客园等地。

**Comment by gitlsl** [comment]
个人感觉这个项目技术水平高，  可以说  曲高和寡，  多数人都不知道这个项目到底干啥的 能做到哪些功能
如果转博客园的话 可以多一些 例子，  估计  用c#写脚本，运行过程中热更新是大家最喜欢的

**Reply by NMSAzulX** [reply]
好的，多谢建议，抽出时间我去博客园写几篇文章。

**Comment by newbe36524** [comment]
go to https://github.com/dotnetcore/Natasha.Docs/issues/1

**Reply by daonidedie** [reply]
切换到英文了，就看不懂了。现在程序员不像以前了，都是抄抄抄的。面向百度和搜索编程的。十多年前都是BBS上交流的多。

**Reply by NMSAzulX** [reply]
> 切换到英文了，就看不懂了。现在程序员不像以前了，都是抄抄抄的。面向百度和搜索编程的。十多年前都是BBS上交流的多。

后面 newBing AI 开放之后逐渐就不用照顾多语言了, 实话说省了不少工作.
